zorder/src/main/scala/org/locationtech/sfcurve/zorder/Z3.scala

```diff
@@ -143,13 +143,13 @@ object Z3 {
           zranges(min, nextOffset, 7, nextLevel)
         } else {
           // bottom out - add the entire range so we don't miss anything
-          mq += IndexRange(octRange.min.z, octRange.max.z, false)
+          mq += IndexRange(octRange.min.z, octRange.max.z, contained = false)
         }
       }
     }
 
     // kick off recursion over the narrowed space
-    zranges(commonPrefix, TOTAL_BITS - commonBits, 0, 0)
+    zranges(commonPrefix, 64 - commonBits, 0, 0)
```

Comment:
Originally it was assumed that the first bit of the longs will always match, since they're not used. But that messed with limiting by precision, as it then would be off by that one bit.
It was also part of some work I did to change the function to be more general, to work with e.g. a z2 index, which might be relevant eventually.